sdk/ml/azure-ai-ml/azure/ai/ml/entities/_featureset/featureset_spec.py

```diff
@@ -0,0 +1,71 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+
+from os import PathLike
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+from marshmallow import INCLUDE
+
+from azure.ai.ml._schema._featureset.featureset_spec_schema import FeaturesetSpecSchema
+from azure.ai.ml._utils.utils import load_yaml
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._util import load_from_dict
+from azure.ai.ml.entities._featurestore_entity.data_column import DataColumn
+
+from .feature import Feature
+from .source_metadata import SourceMetadata
+from .delay_metadata import DelayMetadata
+
+
+class FeaturesetSpec(object):
```

Comment:
Shall we spell out the specification fully ?

Reply:
there is another class called FeaturesetSpecification